tox.ini

```diff
@@ -36,13 +36,13 @@ commands =
 basepython = python3.6
 # pylint==2.1.1 required due to prospector bug, see https://github.com/PyCQA/prospector/pull/309
 deps =
-    prospector==1.1.4
-    pylint==2.1.1
+    prospector==1.1.6.2
+    pylint==2.2.2
```

Comment:
pylint 2.2.2 produces an error.
we can get rid of this dependency, because prospector pins a working pylint version.
This should solve the CI errors for the lint env.

Reply:
I think the issue is [this](https://github.com/PyCQA/prospector/pull/309) with prospector (I pinned pylint to avoid it).

Any views on prospector over flake8 + pylint, I prefer the latter setup so I'm open to changing.

Reply:
https://github.com/PyCQA/prospector/pull/304

I have no feeling either way. Choose what works best currently - it's a constant race anyway.

Reply:
Thanks, I've just removed it in c04e260